cognitedata/reveal
# feat(react-components): add CAD models style caching support

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/

## Description :pencil:

Adds style caching support for CAD models. Also adds styling stress testing in `CadModelContainer` story.

## How has this been tested? :mag:


1. In HighlightNode story and in CadModelContainer story

## Test instructions :information_source:


1. Go to HighlightNode story and click on different mapped nodes
2. Go to CadModelContainer story and click anywhere in the viewer a 10-20 times. You can see new styling groups appearing in the console.

## Checklist :ballot_box_with_check:
